docs/03-pages/01-building-your-application/05-optimizing/10-testing.mdx

```diff
@@ -284,6 +285,8 @@ You can learn more about Playwright and Continuous Integration from these resour
 - [Getting started with Playwright](https://playwright.dev/docs/intro)
 - [Use a development server](https://playwright.dev/docs/test-webserver/)
 - [Playwright on your CI provider](https://playwright.dev/docs/ci)
+- [Ask Playwright (AI trained to answer questions about Playwright)](https://ray.run/ask)
```

Comment:
```suggestion
```

The tool looks great! However, we don't want to link to something thats not from the official Playwright team right now. Thank you for your understanding :pray: